modules/gmake2/gmake2_workspace.lua

```diff
@@ -115,7 +115,14 @@
 --
 
 	function gmake2.projects(wks)
-		_p('PROJECTS := %s', table.concat(p.esc(table.extract(wks.projects, "name")), " "))
+		local prjs = table.extract(wks.projects, "name")
+
+		local escapedPrjs = {}
+		for _,v in ipairs(prjs) do
+			table.insert(escapedPrjs, gmake2.esc(v))
```

Comment:
Why `gmake2.esc` instead of `p.esc`? `p.esc` should already be set to `gmake2.esc`.

Reply:
Hmm, let me check something with the unit test.  It seems to be picking up the global `p.esc` definition rather than the `gmake2.esc` definition.

Reply:
Follow up: Yeah, setting the escaper in the unit test fixed the problem. Updating now.